src/components/headers/Listmaster.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+
+      <!-- Title -->
+      <span class="mdl-layout-title header-title">{{ $route.meta.title || "Sympa" }}</span>
+      <div class="mdl-layout-spacer"></div>
+
+      <!-- Actions -->
+      <button class="mdl-button mdl-js-button mdl-button--icon">
```

Comment:
Maybe move this in a component with a sane lifecycle (and not a global DOM upgrade after each transition)

Reply:
https://github.com/sympa-community/sympa-vue/pull/2#discussion_r120758329